crates/sc-consensus-subspace/src/slot_worker.rs

```diff
@@ -294,6 +298,25 @@ where
             }
         }
 
+        // TODO: This is a workaround for potential root cause of
+        //  https://github.com/subspace/subspace/issues/871, also being discussed in
+        //  https://substrate.stackexchange.com/questions/7886/is-block-creation-guaranteed-to-be-running-after-parent-block-is-fully-imported
+        if maybe_pre_digest.is_some() {
+            let block_number = *parent_header.number() + One::one();
+            let (acknowledgement_sender, mut acknowledgement_receiver) = mpsc::channel(0);
```

Comment:
This dies not seem to be used in the archiver yet, is this for later use?

Reply:
It is, always was. We're not sending a message, but we hold only sender to pause receiver here.